src/Ship.cpp

```diff
@@ -493,10 +493,12 @@ bool Ship::OnDamage(Object *attacker, float kgDamage, const CollisionContact &co
 			if (Pi::rng.Double() < kgDamage)
 				SfxManager::Add(this, TYPE_DAMAGE);
 
-			if (dam < 0.01 * float(GetShipType()->hullMass))
-				Sound::BodyMakeNoise(this, "Hull_hit_Small", 1.0f);
-			else
-				Sound::BodyMakeNoise(this, "Hull_Hit_Medium", 1.0f);
+			if (dam > float(GetShipType()->hullMass / 1000.)) {
```

Comment:
Shouldn't that be a less than?
> `if (dam < float(GetShipType()->hullMass * 0.0001)) {`

Because it's a miniscule amount of damage that you don't want to hear rather than a huge amount of damage?

Reply:
I read it as "If damage is _more_ than 'amount' go in parentheses" and then it choose if play big or medium sound...

Where 'amount' is ship mass in [g]: `dam` is [Kg]*[m/sec]*[m/sec], but is multiplied at the function begin in [g], thus this is checking if the speed is above 1 m/sec.

What I try to avoid is to have that little sound played continuosly even when Ship is more or less firm... Don't know, but trying these things I started hating that sound :D

Reply:
ah I see, I fail at reading comprehension today!